### ISSUE TITLE
Focalpoint another VR-Toolkit

### ISSUE THREAD

**Issue by RoccoGD** [thestonefox/SteamVR_Unity_Toolkit#148]
Here is another Toolkit with an interesting locomotion approach, which i havn't seen before.
It's a drag and drop locomotion with the controller.
And there are some other interesting features like scale objects, so i think it's good for inspiration to take a look.

Focalpoint on Unity Assetstore - https://www.assetstore.unity3d.com/en/#!/content/59625
On Github -> https://github.com/Albert/FocalPointVR

And until we have a proper place to talk just casual about such findings, i have to make an "issue" out of it, which isn't the way to go in the long term. A simple old style forum would do the job i guess.

**Comment by thestonefox** [comment]
Is it worth creating a subreddit for this repo?

**Comment by thestonefox** [comment]
That's what I figured about slack too, I prefer a good forum

**Comment by thestonefox** [comment]
I created this: 
https://www.reddit.com/r/SteamVRUnityToolkit/

**Comment by RoccoGD** [comment]
Yes thats fine. :+1:

**Comment by RoccoGD** [comment]
So we have a reddit Forum now we can close this on here.
Here is a link https://www.reddit.com/r/SteamVRUnityToolkit/comments/4n06bh/vr_toolkits_for_unity/

**Comment by philmennie** [comment]
Hey, I'm kind of new to this so sorry if I'm writing in the wrong place - let me know and I'll move / delete this comment.  I'm currently having issues detecting controller input while using VRTK.  For example, when I have a collision between two objects, I want to be able to detect what buttons are being pressed on the controllers but can't seem to work out how I can do this.  Also, I have implemented the Interact Use functionality but I'm struggling to work out how to make two buttons do different actions.  For example, once I grab an object with the simple pointer I want one button to bring the object closer and another to move it away, but I've only managed to implement one or the other.  Any suggestions?  I've looked everywhere in the docs, examples and Google and can't seem to find anything.  Any help would be MUCH appreciated!  Pulling my hair out here!
